server/src/map.c

```diff
@@ -0,0 +1,90 @@
+/*
+** EPITECH PROJECT, 2024
+** Project
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#include "map.h"
+#include "egg.h"
+
+static void generate_stones(map_t *map, int x, int y)
+{
+    for (int i = 0; i < (map->height * map->width) / 10; i++) {
+        x = rand() % map->width;
+        y = rand() % map->height;
+        get_tile(map, x, y)->resources.mendiane_count += 1;
+    }
+    for (int i = 0; i < (map->height * map->width) / 8; i++) {
+        x = rand() % map->width;
+        y = rand() % map->height;
+        get_tile(map, x, y)->resources.phiras_count += 1;
+    }
+    for (int i = 0; i < (map->height * map->width) / 5; i++) {
+        x = rand() % map->width;
+        y = rand() % map->height;
+        get_tile(map, x, y)->resources.thystame_count += 1;
+    }
+    for (int i = 0; i < (map->height * map->width) / 10; i++) {
+        x = rand() % map->width;
+        y = rand() % map->height;
+        get_tile(map, x, y)->resources.sibur_count += 1;
+    }
+}
+
+void generate_ressources(map_t *map)
+{
+    int x = 0;
+    int y = 0;
+
+    for (int i = 0; i < (map->height * map->width) / 50; i++) {
```

Comment:
same here this for loop shall be extracted

Reply:
same here